changes/next/jmap_contact_new_blobids

```diff
@@ -0,0 +1,13 @@
+Description:
+
+Use mailbox id + message UID as blob id
```

Comment:
Nit: this doesn't make clear for what kind of blobs this change applies. It's JMAP CalendarEvent, Contact and ContactGroup, right?